### ISSUE TITLE
Download got stuck on "Querying info"

### ISSUE THREAD

**Issue by aydogduonur** [KRTirtho/spotube#1896]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

![Screenshot_2024-09-18-16-47-43-870_oss krtirtho spotube-edit](https://github.com/user-attachments/assets/e8292bc1-0519-4192-bbc7-7aff45312336)

It got stuck on this phase. 

### Expected Behavior

It should have _already_ downloaded the music I chose instead of making me wait on "Querying info" stage for hours. 

### Steps to reproduce

https://github.com/user-attachments/assets/3ed71b70-26fa-46cc-ac5f-f34e9c0690af

### Operating System

Android 13

### Spotube version

v3.8.1

### Installation source

Website (spotube.krtirtho.dev)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
Duplicate of #1685

**Comment by aydogduonur** [comment]
You do _not_ understand one thing apparently. If it _is_ a duplicate of an issue and if what you did to fix the issue didn't work, then you _should/have to_ find another workaround. Got it?

**Comment by Kiennguyen0945** [comment]
You can try changing the source of the music. I use an Android phone. First, I erased all Spotube memory and changed the music source from YouTube to another (I currently chose Piped). After that, I signed into my Spotify account and tried to download it again. If the issue still appears, repeat all the steps above.


![Screenshot_20250110-111343](https://github.com/user-attachments/assets/45469254-84e5-4153-bde5-da4260a3190d)
